chalicelib/checks/wrangler_checks.py

```diff
@@ -541,15 +541,15 @@ def change_in_item_counts(connection, **kwargs):
     # add deleted/replaced items
     search_query += '&status=deleted&status=replaced'
     search_resp.extend(ff_utils.search_metadata(search_query, key=connection.ff_keys))
-    search_output = []
     for res in search_resp:
-        # convert type to index name. e.g. ExperimentSet --> experiment_set
-        res_index = convert_camel_to_snake(res['@type'][0])
-        if res_index in diff_counts:
-            diff_counts[res_index]['ES'] = diff_counts[res_index]['ES'] + 1 if 'ES' in diff_counts[res_index] else 1
+
+        # Stick with given type name in CamelCase since we need it to match ES type-name
```

Comment:
This isn't clear from the comment so I'll fix it but the CamelCase-ness comes from the counts page (snovault). We used to have to convert CamelCase to snake_case because snake_case is what we transformed to in snovault. The update caused the symmetric change to now convert snake_case to CamelCase, so the conversion here is now unnecessary.